build/common.jl

```diff
@@ -25,17 +35,41 @@ function remote_url(repo_root::AbstractString, name::AbstractString="origin")
     end
 end
 
-const REPO_PATH = abspath(joinpath(@__DIR__, "..", ".."))
-const PKG_URL = remote_url(REPO_PATH)
+function set_url_scheme(url, scheme)
+    m = match(LibGit2.URL_REGEX, url)
+    if m === nothing
+        throw(ArgumentError("URL is not a valid SCP or HTTP(S) URL: $(url)"))
+    end
+    return LibGit2.git_url(; scheme="https", username=something(m[:user], ""),
+                           host=something(m[:host], ""), port=something(m[:port], ""),
+                           path=something(m[:path], ""))
+end
 
-# Read Project.toml
-const JLL_PATH = joinpath(REPO_PATH, "ray_julia_jll")
-const JLL_PROJECT_TOML = joinpath(JLL_PATH, "Project.toml")
-const JLL_ARTIFACTS_TOML = joinpath(JLL_PATH, "Artifacts.toml")
+# Used to convert `HostPlatform` into something contained in
+# `BinaryBuilder.support_platforms()`
+function supported_platform(p::Platform)
+    support_tags = filter((k, v)::Pair -> k in ("call_abi", "libc"), tags(p))
+    support_tags["julia_version"] = string(Base.thisminor(VERSION))
```

Comment:
By using `Base.thisminor` we ensure that the `julia_version` is always `X.Y.0` which produces a more consistent artifact URL